homework1/1.1/1.1/1.1.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Sort
+{
+    public class SortClass
+    { 
+        public static int[] BubbleSort(int[] array)
+        {
+            int temporary;
+            for (int i = 0; i < array.Length; i++)
+            {
+                for(int j = i + 1; j < array.Length; j++)
+                {
+                    if (array[i] > array[j])
+                    {
+                        temporary = array[i];
+                        array[i] = array[j];
+                        array[j] = temporary;
+                    }
+                }
+            }
+            return array;
+        }
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите количество элементов массива:");
+            var inputString = Console.ReadLine();
+            if (inputString == null)
+            {
+                return;
+            }
+            int countOfElements = int.Parse(inputString);
+            int[] array = new int[countOfElements]; 
```

Comment:
```suggestion
            var array = new int[countOfElements]; 
```
В C# практически никогда не надо один тип писать на одной строке дважды